src/test-helpers.tsx

```diff
@@ -541,6 +541,30 @@ export const createFakeEvent = (extraProps = {}) => {
   };
 };
 
+type CreateFakeKeyboardEventParams = {
+  altKey?: boolean;
+  ctrlKey?: boolean;
+  key: string;
+  metaKey?: boolean;
+  shiftKey?: boolean;
+};
+
+export const createFakeKeyboardEvent = ({
```

Comment:
I was playing around with this to understand why you needed so many type assertions and it looks like you don't need to create a fake event at all. This is working for me:

```diff
diff --git a/src/components/KeyboardShortcuts/index.spec.tsx b/src/components/KeyboardShortcuts/index.spec.tsx
index 3d4e098..60ae453 100644
--- a/src/components/KeyboardShortcuts/index.spec.tsx
+++ b/src/components/KeyboardShortcuts/index.spec.tsx
@@ -5,7 +5,7 @@ import configureStore from '../../configureStore';
 import { RelativePathPosition } from '../../reducers/fileTree';
 import { actions as versionActions } from '../../reducers/versions';
 import {
-  createFakeKeyboardEvent,
+  createKeydownEvent,
   createFakeThunk,
   shallowUntilTarget,
   spyOn,
@@ -100,7 +100,7 @@ describe(__filename, () => {
 
       // TS cannot seem to figure out that `key` is a string.
       // @ts-ignore
-      const event = createFakeKeyboardEvent({ key }) as KeyboardEvent;
+      const event = createKeydownEvent({ key }) as KeyboardEvent;
 
       keydownListener(event);
 
@@ -127,7 +127,7 @@ describe(__filename, () => {
 
     const { keydownListener } = root.instance() as KeyboardShortcutsBase;
 
-    const event = createFakeKeyboardEvent({ key: 'e' }) as KeyboardEvent;
+    const event = createKeydownEvent({ key: 'e' });
 
     keydownListener(event);
 
@@ -149,7 +149,7 @@ describe(__filename, () => {
 
     const { keydownListener } = root.instance() as KeyboardShortcutsBase;
 
-    const event = createFakeKeyboardEvent({ key: 'c' }) as KeyboardEvent;
+    const event = createKeydownEvent({ key: 'c' });
 
     keydownListener(event);
 
@@ -168,10 +168,10 @@ describe(__filename, () => {
 
       const { keydownListener } = root.instance() as KeyboardShortcutsBase;
 
-      const event = createFakeKeyboardEvent({
+      const event = createKeydownEvent({
         key: 'k',
         [modififierKey]: true,
-      }) as KeyboardEvent;
+      });
 
       keydownListener(event);
 
diff --git a/src/test-helpers.tsx b/src/test-helpers.tsx
index 4a67a9d..d4f6817 100644
--- a/src/test-helpers.tsx
+++ b/src/test-helpers.tsx
@@ -549,20 +549,20 @@ type CreateFakeKeyboardEventParams = {
   shiftKey?: boolean;
 };
 
-export const createFakeKeyboardEvent = ({
+export const createKeydownEvent = ({
   altKey = false,
   ctrlKey = false,
   key,
   metaKey = false,
   shiftKey = false,
 }: CreateFakeKeyboardEventParams) => {
-  return {
+  return new KeyboardEvent('keydown', {
     altKey,
     ctrlKey,
     key,
     metaKey,
     shiftKey,
-  };
+  });
 };
 
 /*
```

Reply:
The refactor you mentioned above to create `renderAndTriggerKeyEvent` actually means that only one type assertion remained, but this still seems like a decent improvement, so I've made this change as well.